media/css/pocket/components/home.scss

```diff
@@ -0,0 +1,145 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/base/elements/containers';
+@import '~@mozilla-protocol/core/protocol/css/components/button';
+@import '../utils/variables';
+@import '../includes/platform-nav';
+@import '../includes/platform-footer';
+
+$image-path: '/media/img/pocket';
+
+.pocket-homepage {
+    color: $color-text-primary;
+    font-family: $font-sans;
+
+    .pocket-homepage-header {
+        padding-top: 0;
+
+        .mzp-c-split-media-asset {
+            max-height: 800px;
+            width: auto;
+        }
+    }
+
+    .section-heading {
+        @include text-title-lg;
+        font-family: $font-sans-alt;
+        font-weight: 500;
+    }
+
+    .sub-section-heading {
+        @include font-size(32px);
+        font-weight: 600;
+    }
+
+    .section-lede {
+        @include text-body-lg;
+    }
+
+    .pocket-homepage-picto {
+        h2 {
+            text-align: center;
+            margin-bottom: $spacing-2xl;
+        }
+
+        .mzp-c-picto-heading {
+            @include text-body-lg;
+            font-weight: 500;
+        }
+
+        .mzp-c-picto-image {
+            margin-bottom: $spacing-xs;
+        }
+    }
+
+    .pocket-homepage-premium {
+        @include text-body-lg;
+        padding-top: $layout-sm;
+        padding-bottom: $layout-md;
+        position: relative;
+
+        h2 {
+            font-family: $font-sans-alt;
+            @include text-title-md;
+            font-weight: 500;
+            text-align: center;
+
+            @media #{$mq-md} {
+                @include text-title-sm;
+                text-align: left;
+            }
+        }
+
+        .mzp-u-list-styled {
+            margin-bottom: $spacing-lg;
+            line-height: 26px;
+
+            li {
```

Comment:
**issue (non-blocking):** vertical spacing between list items looks too close to line height

**suggestion:** add margin-bottom to list items to more easily scan separate items